website/docs/guides/customize-schema-alias.md

```diff
@@ -0,0 +1,490 @@
+
+
+## Example use cases
+
+Here are some typical examples we've encountered with dbt users leveraging those 3 macros and different logic. 
+
+:::note
+
+Note that the following examples do not comprehensively cover all the available options. These examples are meant to be templates for you to develop your own behaviors. 
+
+:::
+
+
+
+### 1. Using the custom schema without the concatenation with the target schema in production
+
+
+The most common use case is using the custom schema without concatenating it with the default schema name when in production.
+
+To do so, you can create a new file called `generate_schema_name.sql` under your macros folder with the following code:
+
+```Jinja
+
+{% macro generate_schema_name(custom_schema_name, node) -%}
+
+    {%- set default_schema = target.schema -%}
+    {%- if custom_schema_name is none -%}
+
+        {{ default_schema }}
+
+    {%- elif  env_var('DBT_ENV_TYPE','DEV') == 'PROD' -%}
+        
+        {{ custom_schema_name | trim }}
+
+    {%- else -%}
+
+        {{ default_schema }}_{{ custom_schema_name | trim }}
+
+    {%- endif -%}
+
+{%- endmacro %}
+
+```
+
+This will generate the following outputs for a model called `my_model` with a custom schema of `marketing`, preventing any overlap of objects between dbt runs from different contexts.
+
+
+| Context     |Target database| Target schema | Resulting object               |
+|-------------|:-------------:|:-------------:|:------------------------------:|
+| Developer 1 | dev           | dbt_dev1      |dev.dbt_dev1_marketing.my_model  |
+| Developer 2 | dev           | dbt_dev2      |dev.dbt_dev2_marketing.my_model  |
+| CI PR 123   | ci            | dbt_pr_123    |ci.dbt_pr_123_marketing.my_model|
+| CI PR 234   | ci            | dbt_pr_234    |ci.dbt_pr_234_marketing.my_model|
+| Production  | prod          | analytics     |prod.marketing.my_model         |
+
+
+:::note
+
+We added logic to check if the current dbt run is happening in production or not. This is important, and we explain why in the section on [What not to do](/guides/customize-schema-alias?step=3#what-not-to-do).
+
+:::
+
+
+### 2. Static schemas - Adding developer identities to the front of tables instead of the schema
+
+Occasionally, we run into instances where the security posture of the organization prevents developers from creating schemas and all developers have to develop in a single schema.
+
+In this case, we can: 
+
+- Change `generate_schema_name()` to use a single schema for all developers, even if a custom schema is set.
+- Update `generate_alias_name()` to append the developer alias and the custom schema to the front of the table name in the dev environment.
+    - This method is not ideal, as it can cause long table names, but it will let developers see in which schema the model will be created in production.
+
+```Jina
+
+{% macro generate_alias_name(custom_alias_name=none, node=none) -%}
```

Comment:
also the macro order doesn't match the recommendation:
- `generate_schema_name` comes first in the bullet, however the code example starts with `generate_alias_name`
- same with alias

Reply:
Hiya @T-Dunlap @b-per 

Based on Mirna's feedback, I think I may have done something wrong when following your notion doc for this section. Can you advise on the correct macro if you have a minute please?

Kind Regards